help/destinations/catalog/crm/salesforce.md

```diff
@@ -235,12 +254,17 @@ To validate that you have correctly set up the destination, follow the steps bel
 ![Platform UI screenshot example showing Segment.](../../assets/catalog/crm/salesforce/segment.png)
 
 1. Finally, log in to the Salesforce website and validate if the profiles from the segment have been added or updated.
+
+    #### Working with Contacts
```

Comment:
Same comment as further above about indenting #### sections. Please use bold if the context does not allow you to create a new #### section (e.g. since this is here within a numbered list, I do not suggest creating new #### sections and would advise to use bold formatting instead.)